src/ol/layer/Base.js

```diff
@@ -48,6 +48,9 @@ import {clamp} from '../math.js';
  * @property {BackgroundColor} [background] Background color for the layer. If not specified, no background
  * will be rendered.
  * @property {Object<string, *>} [properties] Arbitrary observable properties. Can be accessed with `#get()` and `#set()`.
+ * @property {boolean} [startDeclutterGroup] Signals to the renderer to start a new declutter group, which causes previous vector(tile)
+ * layers with the declutter option set true to declutter before rendering the current layer. May be set on multiple layers to manage
+ * multiple declutter efforts.
```

Comment:
The new option won't work on layer groups (because they are not included in the `layerStatesArray`), so it should not be added on `Group`'s base here either.

Reply:
Whoops, carry-over from the previous approach, thanks for catching.

Reply:
I was adding the new property to Layer.test.js and need to clarify that `startDeclutterGroup` should only be a private member of `Layer` and not included in the `layerState` defined in `Base.js` --is that correct? I'm a little fuzzy on the role of the state.